test/stresstest/README.md

```diff
@@ -0,0 +1,13 @@
+The stresstest can be invoked by running: ''' python3 stresstest.py '''
+Before starting the stresstest run either: ''' python3 broker.py ''' or ''' python3 server.py '''
+For the configuration which kind of stresstest is executed , you can specify a config file like [config.ini](config.ini).
```

Comment:
How do I specify a config file?
When I read this I expect a CLI option that allows me to specify a config file.
Also, see my comment hereafter for config.py but it seems the config file is expected to be in the current working directory in order to work. That should be mentioned if it is expected here.

Reply:
Good point